src/EditorFeatures/VisualBasic/Wrapping/ChainedExpression/VisualBasicChainedExpressionWrapper.vb

```diff
@@ -0,0 +1,20 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.Editor.VisualBasic.Formatting.Indentation
+Imports Microsoft.CodeAnalysis.Editor.Wrapping.ChainedExpression
+Imports Microsoft.CodeAnalysis.VisualBasic.Indentation
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.Wrapping.ChainedExpression
+    Friend Class VisualBasicChainedExpressionWrapper
+        Inherits AbstractChainedExpressionWrapper(Of NameSyntax, ArgumentListSyntax)
+
+        Public Sub New()
+            MyBase.New(VisualBasicIndentationService.WithoutParameterAlignmentInstance, VisualBasicSyntaxFactsService.Instance)
+        End Sub
+
+        Protected Overrides Function GetNewLineBeforeOperatorTrivia(newLine As SyntaxTriviaList) As SyntaxTriviaList
+            Return newLine.InsertRange(0, {SyntaxFactory.WhitespaceTrivia(" "), SyntaxFactory.LineContinuationTrivia("_")})
```

Comment:
> SyntaxFactory.WhitespaceTrivia(" ")

❔  Is this equivalent to `SyntaxFactory.ElasticSpace`?

Reply:
Good question.  Tried it and everything went off the rails.  The VB formatter started emitting newlines *before* the elasticspace.  No clue why.  Don't want to spend time figuring out :D